cpp/cmake/thirdparty/CUDF_GetRMM.cmake

```diff
@@ -20,14 +20,18 @@ function(find_and_configure_rmm VERSION)
         return()
     endif()
 
+    if(${VERSION} MATCHES [=[([0-9]+)\.([0-9]+)\.([0-9]+)]=])
+        set(MAJOR_AND_MINOR "${CMAKE_MATCH_1}.${CMAKE_MATCH_2}")
+    endif()
```

Comment:
To be extra safe we might want to do the following in-case at some point the `.00` get dropped from the input

```cmake
if(${VERSION} MATCHES [=[([0-9]+)\.([0-9]+)\.([0-9]+)]=])
  set(MAJOR_AND_MINOR "${CMAKE_MATCH_1}.${CMAKE_MATCH_2}")
else()
  set(MAJOR_AND_MINOR "${VERSION}")
endif()
```